src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,84 @@
-import React from 'react';
+import React, { useState } from 'react';
+import classNames from 'classnames';
+import { MovieCard } from '../MovieCard';
 import './FindMovie.scss';
 
-export const FindMovie: React.FC = () => {
+import { getMovie } from '../../api';
+import { Movie } from '../../types/Movie';
+
+type Props = {
+  addMovie: (movie: Movie) => void;
+  moviesList: Movie[]
+};
+
+export const FindMovie: React.FC<Props> = ({ addMovie, moviesList }) => {
+  const [titleSearchQuery, setSearchField] = useState('');
+  const [movie, setMovie] = useState<Movie | null>(null);
+  const [isError, setError] = useState(false);
+  const [isLoading, setLoading] = useState(false);
+
+  const handleSearchFieldChange
+  = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const { value } = event.target;
+
+    setError(false);
+    setSearchField(value);
+  };
+
+  const createMovie = async (input = titleSearchQuery) => {
+    if (isError) {
+      setError(false);
+    }
+
+    setLoading(true);
+
+    const movieData = await getMovie(input);
```

Comment:
```suggestion
  const createMovie = async () => {
    if (isError) {
      setError(false);
    }

    setLoading(true);

    const movieData = await getMovie(titleSearchQuery);
```

It should work, so you can use `onClick={createMovie}`